tests/meta/upload/conftest.py

```diff
@@ -88,3 +93,31 @@ def analysis(analysis_store, case_id, timestamp):
 def fixture_genotype_analysis_sex() -> dict:
     """Return predicted sex per sample_id."""
     return {"ADM1": "male", "ADM2": "male", "ADM3": "female"}
+
+
+@pytest.fixture(name="mip_case")
+def fixture_mip_case(dna_mip_context, helpers) -> models.Family:
```

Comment:
use `from` all the way to `models` and then just import `Family` in the import
```suggestion
def fixture_mip_case(dna_mip_context, helpers) -> Family:
```
Have you checked if we have a similar fixture elsewhere that you can reuse to try to keep this DRY